stories/code/code.stories.js

```diff
@@ -24,13 +24,21 @@ spec:
   installableScopes:
     - Cluster
   extensionConfig:
-    extensionManagerServiceUrl: https://extension-manager.{{ .Values.landscapeDomain }}/graphql
+    extensionManagerServiceUrl: https://extension-manager-name.{{ .Values.landscapeDomainName }}/graphql
 </code></pre>
 `;
 
-primary.storyName = 'Code';
+primary.storyName = 'Code Block';
 primary.parameters = {
     docs: {
         storyDescription: 'Code example'
```

Comment:
Maybe add more info here. It doesn't become clear what's the purpose of this modifier class.